quickstart.md

```diff
@@ -161,253 +167,45 @@ NAME            HANDLER         AGE
-EOT
-```
-
-Be sure to replace `[REGISTRY_URL]` with the desired registry URL of the encrypted image generated in previous step, replace `[RUNTIME_CLASS]` with kata runtime class for CC HW.
-
-Then configure `/opt/confidential-containers/share/defaults/kata-containers/configuration-<RUNTIME_CLASS_SUFFIX>.toml` to add `agent.aa_kbc_params=cc_kbc::<KBS_URI>` to kernel parameters. Here `RUNTIME_CLASS_SUFFIX` is something like `qemu-tdx`, `KBS_URI` is the address of Key Broker Service in KBS cluster like `http://123.123.123.123:8080`.
-
-Deploy encrypted image as a workload:
-
-```shell
-kubectl apply -f encrypted-image-test-busybox.yaml
-```
+certain platforms may require additional configuration to enable confidential computing. For example, a specific 
+host kernel or firmware may be required. See the [guides](./guides/) for more information.
+
+## Using CoCo
+
+Below is a brief summary and description of some of the CoCo use cases and features:
+
+- **Container Launch with Only Memory Encryption (No Attestation)** - Launch a container with memory encryption 
+- **Container Launch with Encrypted Image** - Launch an encrypted container by proving the workload is running 
+  in a TEE in order to retrieve the decryption key
+- **Container Launch with Image Signature Verification** - Launch a container and verify the authenticity and 
+  integrity of an image by proving the workload is running in a TEE
+- **Sealed secret** - Implement wrapped kubernetes secrets that are confidential to the workload owner and are
+  automatically decrypted by proving the workload is running in a TEE
+- **Ephemeral Storage** - Temporary storage that is used during the lifecycle of the container but is cleared out 
+  when a pod is restarted or finishes its task. At the moment, only ephemeral storage of the container itself is 
+  supported and it has to be explicityly configured.
+- **Authenticated Registries** - Create secure container registries that require authentication to access and manage container 
+  images that ensures that only trusted images are deployed in the Confidential Container. The host must have access 
+  to the registry credentials.
+- **Secure Storage** - Mechanisms and technologies used to protect data at rest, ensuring that sensitive information 
+  remains confidential and tamper-proof.
+- **Peer Pods** - Enable the creation of VMs on any environment without requiring bare metal servers or nested 
+  virtualization support. More information about this feature can be found [here](https://github.com/confidential-containers/cloud-api-adaptor/tree/main).
+  
+
+  The CoCo use cases are implemented differently for each TEE, and they are described in each corresponding 
+[guide](./guides) section. To get started using CoCo without TEE hardware, follow the CoCo-dev guide below:
+
+- [CoCo-dev](./guides/coco-dev.md)
+- [SEV(-ES)](./guides/sev.md)
+- [SNP](./guides/snp.md)
+- TDX
+- [SGX](./guides/enclave-cc.md)
+- Secure Enclave
```

Comment:
I think Secure Enclave either meant IBM Secure Execution or Intel SGX. Either way I think this bullet goes away unless I'm forgetting someone's TEE?

```suggestion
```

Reply:
yes, I believe it was SGX, just removed and updated.